app/seeders/common.yml

```diff
@@ -280,17 +280,13 @@ project_roles:
       - :view_work_packages
       - :view_calendar
       - :comment_news
-      - :browse_repository
-      - :view_changesets
```

Comment:
These are being removed from all seeds taking place as of the moment the PR is merged in. Is it okay if the non member and anonymous roles seeded prior to this keep the two removed permissions? Is a migration not needed to remove them from already seeded instances?

Reply:
I would leave them as they are. We cannot know at this point whether the roles have the permissions out of a conscious decision by some user or it is just the result of seeding.